x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/aggregation/blockhash/BytesRefBlockHash.java

```diff
@@ -87,6 +88,9 @@ IntVector add(BytesRefVector vector) {
     }
 
     IntBlock add(BytesRefBlock block) {
+        if (block instanceof OrdinalBytesRefBlock ordinalsBlock) {
```

Comment:
Could you make an `asOrdinals` instead? I like the `asVector` pattern we have now. It's nice looking at `Block` and seeing "oh, you can view this as a Vector sometimes". And this'd show the same thing.

Reply:
Yes, I added `asOrdinals` in https://github.com/elastic/elasticsearch/pull/108165/commits/a455c55a9d73f38e155afbbb13cd938d31d69a1b.

Reply:
:heart: